### ISSUE TITLE
conflicting values for variable 'line'  @ merging step

### ISSUE THREAD

**Issue by agrouaze** [umr-lops/xsar_slc#108]
```python
Traceback (most recent call last):
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/bin/L1B_xspectra_IW_SLC_IFR", line 8, in <module>
    sys.exit(main())
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/lib/python3.9/site-packages/xsarslc/scripts/L1B_xspectra_IW_SLC_IFR.py", line 169, in main
    generate_IW_L1Bxspec_product(slc_iw_path=slc_iw_path, output_filename=output_filename,
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/lib/python3.9/site-packages/xsarslc/scripts/L1B_xspectra_IW_SLC_IFR.py", line 92, in generate_IW_L1Bxspec_product
    one_subswath_xspectrum_dt = proc.compute_subswath_xspectra(dt, polarization=polarization.upper(),
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/lib/python3.9/site-packages/xsarslc/processing/xspectra.py", line 38, in compute_subswath_xspectra
    intra_xs = compute_IW_subswath_intraburst_xspectra(dt, polarization=polarization, tile_width=tile_width_intra,
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/lib/python3.9/site-packages/xsarslc/processing/xspectra.py", line 185, in compute_IW_subswath_intraburst_xspectra
    xspectra = xr.combine_by_coords([x.drop(['tile_sample', 'tile_line']).expand_dims('burst') for x in xspectra], combine_attrs='drop_conflicts')
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/lib/python3.9/site-packages/xarray/core/combine.py", line 993, in combine_by_coords
    return merge(
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/lib/python3.9/site-packages/xarray/core/merge.py", line 1025, in merge
    merge_result = merge_core(
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/lib/python3.9/site-packages/xarray/core/merge.py", line 757, in merge_core
    variables, out_indexes = merge_collected(
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/lib/python3.9/site-packages/xarray/core/merge.py", line 302, in merge_collected
    merged_vars[name] = unique_variable(
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/lib/python3.9/site-packages/xarray/core/merge.py", line 156, in unique_variable
    raise MergeError(
xarray.core.merge.MergeError: conflicting values for variable 'line' on objects to be combined. You can skip this check by specifying compat='override'.
INFO:prun:[2023-03-30T20:17:58Z] End of processing for input line 2 : '/home/datawork-cersat-public/project/mpc-sentinel1/data/esa/sentinel-1a/L1/IW/S1A_IW_SLC__1S/2021/030/S1A_IW_SLC__1SDV_20210130T052913_20210130T052940_036363_04446A_02D5.SAFE/measurement/s1a-iw3-slc-vv-20210130t052914-20210130t052939-036363-04446a-006.tiff tiles17.7km 1.4g'. EXIT_CODE=1
```

**Comment by agrouaze** [comment]
lastest processing gives about 15% of impacted products

**Comment by agrouaze** [comment]
we have also the same error but for merging the interburst xspectra:
```python
Traceback (most recent call last):
  File "/opt/conda-envs/dev/bin/L1B_xspectra_IW_SLC_IFR", line 8, in <module>
    sys.exit(main())
  File "/home1/datahome/agrouaze/sources/git/xsar_slc/xsarslc/scripts/L1B_xspectra_IW_SLC_IFR.py", line 170, in main
    generate_IW_L1Bxspec_product(slc_iw_path=slc_iw_path, output_filename=output_filename,
  File "/home1/datahome/agrouaze/sources/git/xsar_slc/xsarslc/scripts/L1B_xspectra_IW_SLC_IFR.py", line 79, in generate_IW_L1Bxspec_product
    one_subswath_xspectrum_dt = proc.compute_subswath_xspectra(dt, polarization=polarization.upper(),
  File "/home1/datahome/agrouaze/sources/git/xsar_slc/xsarslc/processing/xspectra.py", line 54, in compute_subswath_xspectra
    inter_xs = compute_IW_subswath_interburst_xspectra(dt, polarization=polarization, tile_width=tile_width_inter,
  File "/home1/datahome/agrouaze/sources/git/xsar_slc/xsarslc/processing/xspectra.py", line 264, in compute_IW_subswath_interburst_xspectra
    xspectra = xr.combine_by_coords([x.drop(['tile_sample', 'tile_line']).expand_dims('burst') for x in xspectra], combine_attrs='drop_conflicts')
  File "/opt/conda-envs/dev/lib/python3.10/site-packages/xarray/core/combine.py", line 993, in combine_by_coords
    return merge(
  File "/opt/conda-envs/dev/lib/python3.10/site-packages/xarray/core/merge.py", line 1025, in merge
    merge_result = merge_core(
  File "/opt/conda-envs/dev/lib/python3.10/site-packages/xarray/core/merge.py", line 757, in merge_core
    variables, out_indexes = merge_collected(
  File "/opt/conda-envs/dev/lib/python3.10/site-packages/xarray/core/merge.py", line 302, in merge_collected
    merged_vars[name] = unique_variable(
  File "/opt/conda-envs/dev/lib/python3.10/site-packages/xarray/core/merge.py", line 156, in unique_variable
    raise MergeError(
xarray.core.merge.MergeError: conflicting values for variable 'line' on objects to be combined. You can skip this check by specifying compat='override'.
```

replacing 
`xr.combine_by_coords([x.drop(['tile_sample', 'tile_line','line','sample']).expand_dims('burst') for x in xspectra], combine_attrs='drop_conflicts')`

by

`xr.combine_by_coords([x.drop(['tile_sample', 'tile_line','line','sample','longitude','latitude']).expand_dims('burst') for x in xspectra], combine_attrs='drop_conflicts')`

seems to fix the issue.

**Comment by lanougue** [comment]
solved in #110
